pkg/kubelet/kubelet.go

```diff
@@ -2183,6 +2188,28 @@ func (kl *Kubelet) reconcileCBR0(podCIDR string) error {
 	return kl.shaper.ReconcileInterface()
 }
 
+func (kl *Kubelet) setHairpin() {
+	const dir = "/sys/devices/virtual/net/cbr0/brif"
```

Comment:
We can avoid hard-coding cbr0 by going broader and setting `/sys/devices/virtual/net/*/brport/hairpin_mode` - the sad truth is that hard-coding cbr0 is not right, and while we do it in some places, we REALLY need to not make it worse.

The risk being that we get some veth interfaces that we don't want, but that seems less bad to me.